sycl/test-e2e/Matrix/runtime_query.cpp

```diff
@@ -0,0 +1,63 @@
+// REQUIRES: gpu-intel-pvc
```

Comment:
Is it possible to make the test run on both PVC and SPR, query the device in the test and based on the result (PVC vs. SPR), check the expected combinations.

Reply:
We don't have SPR in CI, so it should be useless

Reply:
We do test on SPR internally.

Reply:
Yeah, done